clang-tools-extra/clang-doc/Representation.cpp

```diff
@@ -481,9 +481,9 @@ ClangDocContext::ClangDocContext(tooling::ExecutionContext *ECtx,
                                  StringRef RepositoryUrl,
                                  StringRef RepositoryLinePrefix, StringRef Base,
                                  std::vector<std::string> UserStylesheets,
-                                 bool FTimeTrace)
+                                 bool KeepJSON, bool FTimeTrace)
```

Comment:
does this actually need to be in the context? you have a `cl::opt`, right? it doesn't have to be a static variable, and then you could use it from other TUs. This is pretty common in LLVM.

Alternatively, the `cl::opt` could just be in the mustache generator, since its the only generator that uses it.